packages/contentful/src/crn/queries/tutorials.queries.ts

```diff
@@ -43,6 +49,74 @@ export const tutorialsContentQueryFragment = gql`
         }
       }
     }
+    authorsCollection(limit: 100) {
```

Comment:
100 seems high, we probably want to reduce it so the query complexity is not too high.

Reply:
I could set the limit at 10 and increase if needed.